zenodo/modules/redirector/views.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015, 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Zenodo redirector."""
+
+from __future__ import absolute_import, print_function
+
+from flask import Blueprint, redirect, url_for
+from invenio_collections.models import Collection
+
+blueprint = Blueprint(
+    'zenodo_redirector',
+    __name__,
+    template_folder='templates',
+    static_folder='static',
+)
+
+
+# https://zenodo.org/collection/user-<id>
+# https://sandbox.zenodo.org/communities/<id>/
+@blueprint.route('/collection/user-<id>')
+def community_redirect(id):
+    """Redirect from the legacy community details to the new one."""
+    redirect(url_for('invenio_communities.detail', community_id=id))
+
+
+# https://zenodo.org/search?ln=en&cc=user-<id>&p=<query>&action_search=
+# https://zenodo.org/communities/<id>/search?q=<query>
+@blueprint.route('/search')
```

Comment:
@JavierDelgadoFernandez FYI I'm not sure if this will work, since it is colliding with the main search endpoint: (e.g. https://sandbox.zenodo.org/search?page=1&size=20&q=). Let me know if it works, otherwise we can discuss other possible solutions.